src/Tribe/Aggregator/Tabs/New.php

```diff
@@ -0,0 +1,109 @@
+<?php
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+class Tribe__Events__Aggregator__Tabs__New extends Tribe__Events__Aggregator__Tabs__Abstract {
+	/**
+	 * Static Singleton Holder
+	 *
+	 * @var self|null
+	 */
+	private static $instance;
+
+	/**
+	 * Static Singleton Factory Method
+	 *
+	 * @return self
+	 */
+	public static function instance() {
+		if ( empty( self::$instance ) ) {
+			self::$instance = new self;
+		}
+
+		return self::$instance;
+	}
+
+	public function __construct() {
+		// Setup Abstract hooks
+		parent::__construct();
+
+		// Configure this tab ajax calls
+		add_action( 'wp_ajax_tribe_ea_dropdown_csv_content_type', array( $this, 'ajax_csv_content_type' ) );
+		add_action( 'wp_ajax_tribe_ea_dropdown_csv_files', array( $this, 'ajax_csv_files' ) );
+		add_action( 'wp_ajax_tribe_ea_dropdown_origins', array( $this, 'ajax_origins' ) );
+
+		// We need to enqueue Media scripts like this
+		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_media' ) );
+	}
+
+	public function enqueue_media() {
+		if ( ! $this->is_active() ) {
+			return;
+		}
+
+		wp_enqueue_media();
+	}
+
+	public $priority = 10;
+
+	public function is_visible() {
+		return true;
+	}
+
+	public function get_slug() {
+		return 'new';
+	}
+
+	public function get_label() {
+		return esc_html__( 'New Import', 'the-events-calendar' );
+	}
+
+	public function ajax_csv_content_type() {
```

Comment:
So, currently I think these new methods would fit a lot better on the Origin Classes, I mean, these are getters that I think we shouldn't be exposing on the Tab itself but on the actual CSV handler.